src/openrct2/title/TitleSequence.h

```diff
@@ -38,24 +38,45 @@ struct TitleCommand
     };
 };
 
+struct TitleSequenceParkHandle
+{
+    const utf8* HintPath;
+    void* Stream;
+};
+
 struct TitleSequence
 {
+    explicit TitleSequence(const utf8* path);
+    TitleSequence() = default;
+    ~TitleSequence() = default;
+
+    bool Save();
+    bool AddPark(const utf8*, const utf8* name);
+    bool RenamePark(size_t index, const utf8* name);
+    bool RemovePark(size_t index);
+    std::string LegacyScriptWrite();
+    void SwapConsecutiveCommands(size_t index);
+
+    // TODO: TitleSequenceParkHandle should also be migrated to RAII
+    TitleSequenceParkHandle* GetParkHandle(size_t index);
+    void CloseParkHandle(TitleSequenceParkHandle* handle);
+
+    size_t NumSaves() const
+    {
+        return Saves.size();
+    }
+    size_t NumCommands() const
+    {
+        return Commands.size();
+    }
+
     std::string Path;
     std::string Name;
 
     std::vector<TitleCommand> Commands;
-    size_t NumCommands() const { return Commands.size(); }
-
     std::vector<std::string> Saves;
-    size_t NumSaves() const { return Saves.size(); }
 
-    bool IsZip;
-};
-
-struct TitleSequenceParkHandle
-{
-    const utf8* HintPath;
-    void* Stream;
+    bool IsZip = false;
```

Comment:
```suggestion
    bool IsZip {false};
```

Maybe use uniform initialization?

(This is on commit https://github.com/OpenRCT2/OpenRCT2/pull/12815/commits/b0f7e8d558e04b4736b41257400ceab4cd73a38b)